### PR TITLE
[plugin.audio.weareone] v1.0.0

### PR DESCRIPTION
**Pull request by fossdd**
### Description
[**WeAreOne Radio Family**](https://weareone.fm/): This adds Streams of every Radio of the family like TechnoBase.FM and HouseTime.FM

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.weareone/addon.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Play Audio Streams by the WeAreOne Family</summary>
+        <description lang="en_GB">WeAreOne is a radio sender family. It contains popular techno and house radios like TechnoBase.FM and HouseTime.FM.</description>
+        <language></language>
+        <platform>all</platform>
+        <license>unlicense</license>
+        <forum></forum>
+        <website></website>
+        <email>fossdd@tutanota.com</email>
+        <source></source>
```
Comment (enen92) [inline plugin.audio.weareone/addon.xml]: Please fill the upstream repo location

File: plugin.audio.weareone/addon.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        <news></news>
```
Comment (enen92) [inline plugin.audio.weareone/addon.xml]: Drop if empty

File: plugin.audio.weareone/addon.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        <import addon="script.module.routing" version="0.2.0"/>
+
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="main.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Play Audio Streams by the WeAreOne Family</summary>
+        <description lang="en_GB">WeAreOne is a radio sender family. It contains popular techno and house radios like TechnoBase.FM and HouseTime.FM.</description>
+        <language></language>
```
Comment (enen92) [inline plugin.audio.weareone/addon.xml]: Drop if empty

File: plugin.audio.weareone/addon.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        <platform>all</platform>
+        <license>unlicense</license>
+        <forum></forum>
```
Comment (enen92) [inline plugin.audio.weareone/addon.xml]: Drop if empty

File: plugin.audio.weareone/addon.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <website></website>
```
Comment (enen92) [inline plugin.audio.weareone/addon.xml]: Drop if empty

**Review by enen92 — CHANGES_REQUESTED** [review]

File: plugin.audio.weareone/addon.xml (hunk 2 above)
Comment (fossdd) [reply plugin.audio.weareone/addon.xml]: should the new commit also have the same name: `[plugin.audio.weareone] v1.0.0`?

**Comment by enen92** [comment]
Preferably squash them. Otherwise just commit and I'll squash before merging (if Travis is green)

**Comment by fossdd** [comment]
Okay, i've fixed all travis issues

**Review by enen92 — APPROVED** [review]

**Comment by fossdd** [comment]
Thank You!

**Pull request merged**